src/test/java/io/lettuce/core/commands/StringCommandIntegrationTests.java

```diff
@@ -238,4 +241,39 @@ void time() {
         Long.parseLong(time.get(0));
         Long.parseLong(time.get(1));
     }
-}
+
+    @Test
+    void strAlgo() {
```

Comment:
Please split this method into multiple test methods so we can test each scenario individually.

Reply:
fixed at last commit.